### PR TITLE
A couple more examples

### PR DESCRIPTION
**Pull request by ColinEberhardt**
A couple more examples, plus a bugfix to the range utility function.

screen shot 2015-11-04 at 21 40 47
screen shot 2015-11-04 at 21 40 57

### CODE REVIEW

**Comment by jdunkerley** [comment]
Both really nice examples @ColinEberhardt.

All LGTM

Out of interest why did you choose to reorder the existing examples?

File: src/util/scale.js [hunk 1]
```diff
@@ -5,8 +5,30 @@ export function isOrdinal(scale) {
...
+    // range set on the scale. The logic below detects the inverted case.
+    //
+    // The d3 code that tackles the same issue doesn't have to deal with the inverted case.
+    var range = scale.range();
+    var extent = scale.rangeExtent();
+    if (range.length <= 1) {
+        // we cannot detect the inverted case if the range (and domain) has
+        // a single item in it.
+        return extent;
+    } else {
```
Comment (chrisprice) [inline src/util/scale.js]: Minor: any reason for the else?

**Comment by chrisprice** [comment]
:+1: Just the one minor comment. Also interested in the answer to @jdunkerley's question.

**Comment by ColinEberhardt** [comment]
> Out of interest why did you choose to reorder the existing examples?

Good question ;-) I'd vaguely considered ordering the examples by complexity, but as we probably don't want too many of them - and you can easily gauge their complexity by a quick glance at their thumbnails it doesn't matter much.

I can change the order back if you two want? However, I think this page needs a bit of work anyhow. I was going to add one or two more examples then take a look at this.

File: src/util/scale.js (hunk 1 above)
Comment (ColinEberhardt) [inline src/util/scale.js]: Before launching into a lengthy reply can I just check ... Are you asking about the construction / cosmetics of the code? Or does your question actually relate to the technique I employing to derive the range?

Comment (chrisprice) [inline src/util/scale.js]: It was stylistic. I don't feel strongly...

**Comment by chrisprice** [comment]
Build failed, @bjedrzejewski's eslint no-shadow changes I think

**Comment by ColinEberhardt** [comment]
> Build failed, @bjedrzejewski's eslint no-shadow changes I think

Yeah, that caught me out with one of the other PRs. Time to rebase

**Pull request merged**